exts/rag_bge_small_en_v15/src/lib.rs

```diff
@@ -23,6 +24,10 @@ use tokio::{
 use tokio_stream::wrappers::UnixListenerStream;
 use tonic::{transport::Server, Request, Response, Status};
 
+include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+const _CHECK_SIG: unsafe extern "C" fn() -> bool = PostmasterIsAliveInternal;
```

Comment:
Is this necessary? If the signature of PostmasterIsAliveInternal changes, wouldn't you get a compilation error even without this?